### PR TITLE
Use cozy-client-js@0.3.13 destroyByID ifMatch option

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: cli/src/remote/index.js [hunk 1]
```diff
@@ -257,7 +257,8 @@ export default class Remote implements Side {
       log.info({path}, 'Deleting folder from the Cozy trash...')
       // FIXME: Would it make sense to have an ifMatch option in destroyById()?
```
Comment (sebn) [inline cli/src/remote/index.js]: Remove this comment ? :smiley:

**Comment by sebn** [comment]
Also CI is broken 😛

**Comment by sebn** [comment]
(looks like the failing test is `test/integration/v3/move.js`)

**Comment by aenario** [comment]
test passed, adding commit to remove the comment. Ready to merge !

**Comment by sebn** [comment]
The disk-usage branch is not covered by a unit test, not really an issue.

👍

**Pull request merged**